views/mouth-ulcers/main-content.md

```diff
@@ -6,22 +6,37 @@ Mouth ulcers need time to heal and there’s no quick fix. Avoiding things that
 - reduce pain
 - help to prevent them returning
 
-### Do:
-
-- use a soft-bristled toothbrush
-- drink cool drinks through a straw
-- stick to softer foods - such as milk, yoghurt and mashed potato
-- get regular dental check-ups
-- eat a healthy, balanced diet
-
-### Don’t:
-
-- eat very spicy, salty or acidic food - such as curries, salted peanuts or citrus fruits
-- eat hard, crunchy food - such as toast, crisps or sweets
-- drink very hot or acidic drinks - such as coffee, fruit juice or fizzy drinks
-- use chewing gum
-- use toothpaste containing sodium lauryl sulphate
-
+<section class="panel panel--binary">
+  <article class="panel__column">
+    <div class="panel__header">
+      <h3>Do</h3>
+    </div>
+    <div class="panel__content">
+      <ul class="list-check">
+        <li>use a soft-bristled toothbrush</li>
+        <li>drink cool drinks through a straw</li>
+        <li>stick to softer foods - such as milk, yoghurt and mashed potato</li>
+        <li>get regular dental check-ups</li>
+        <li>eat a healthy, balanced diet</li>
+      </ul>
+    </div>
+  </article>
+  <article class="panel__column">
+    <div class="panel__header">
+      <h3>Don't</h3>
+    </div>
+    <div class="panel__content">
+      <ul class="list-check">
```

Comment:
Not sure whether you mean the wrapper div or the ul? 

Can you add <div> in markdown? If not, anything wrapped in a <div> is treated as html. 

What am I missing?